### DISCUSSION TITLE
也许是第一个利用swow写的完整的项目~

### DISCUSSION THREAD

**Discussion by AuroraYolo** [swow/swow#77] [Show and tell]
🚀 🐟 给👴冲!  https://github.com/Hyperf-Glory/Serendipity-job  Swow加油啊!

**Comment by cexll** [comment]
> 🚀 🐟 给👴冲! https://github.com/Hyperf-Glory/Serendipity-job Swow加油啊!

链接失效了

**Reply by AuroraYolo** [reply]
现在是私人仓库，还没有开源

**Reply by AuroraYolo** [reply]
https://github.com/SerendipitySwow/Serendipity-job.   @cexll
